src/frontend/src/icp/components/transactions/IcNoIndexPlaceholder.svelte

```diff
@@ -1,21 +1,28 @@
 <script lang="ts">
+	import IconAlertTriangle from '$lib/components/icons/lucide/IconAlertTriangle.svelte';
 	import IconSearchClose from '$lib/components/icons/lucide/IconSearchClose.svelte';
 	import RoundedIcon from '$lib/components/ui/RoundedIcon.svelte';
 	import { i18n } from '$lib/stores/i18n.store';
 	import { replaceOisyPlaceholders } from '$lib/utils/i18n.utils';
+
+	export let issueType: 'missing' | 'not-working' = 'missing';
```

Comment:
`placeholderType` maybe? Just thinking that this component is a placeholder so not strictly related to why if that makes sense?

Reply:
makes sense, changed